src/test/java/com/mael/ttt/GameTest.java

```diff
@@ -1,74 +1,65 @@
 package com.mael.ttt;
 
+import com.mael.ttt.players.HumanPlayer;
 import com.mael.ttt.ui.SpyConsole;
 import com.mael.ttt.ui.UserInterface;
 import org.junit.Before;
 import org.junit.Test;
 
+import java.util.Arrays;
+
 import static org.hamcrest.CoreMatchers.containsString;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertThat;
 
 public class GameTest {
 
-    private Game game;
-    Board board;
-    private SpyConsole spy;
     private int size;
+    private Board board;
+    private SpyConsole spy;
+    private UserInterface gameUI;
+    private Game game;
+
     private String playerMark;
     private String opponentMark;
 
     @Before
     public void setUp() {
-        size  = 3;
-        board = new Board(size);
-        spy   = new SpyConsole();
-        game  = new Game(board, new UserInterface(spy));
+        size   = 3;
```

Comment:
What value do you get from extracting most of these temporary variables into fields? 
I'd prefer to be able to lookup at the 2-3 object that really matter for this test (the Game and the Spy?) in an instant rather than having to figure out which ones are critical and which are dependencies that I just need.